### PR TITLE
add_dump_redis_command

### PR DESCRIPTION
**Pull request by jomenxiao**
dump all namespace commands to separate files, which rotate by size.

We use titan as a platform for all teams.
Sometimes we want to know 
- which command with params is high latency, and hurt the platform
- for audits

### CODE REVIEW

File: bin/titan/main.go [hunk 1]
```diff
@@ -23,16 +23,19 @@ import (
...
 func main() {
 	var showVersion bool
 	var confPath string
 	var pdAddrs string
+	var dumpRedisCommand string
 
 	flag.BoolVar(&showVersion, "v", false, "Show Version")
 	flag.StringVar(&confPath, "c", "conf/titan.toml", "conf file path")
 	flag.StringVar(&pdAddrs, "pd-addrs", "", "pd cluster addresses")
+	flag.StringVar(&dumpRedisCommand, "dump", "", "dump redis command and duration to file")
```
Comment (arthurkiller) [inline bin/titan/main.go:38]: maybe `bool` is better

File: bin/titan/main.go [hunk 2]
```diff
@@ -52,6 +55,9 @@ func main() {
 		fmt.Println("Warning: Titan is running in memory mode, all the data would be cleared after the process exit.")
 		fmt.Println("The memory mode can only be used for experience, configure the tikv.pd-addrs to use the TiKV as a backend")
 	}
+	if dumpRedisCommand != "" {
```
Comment (arthurkiller) [inline bin/titan/main.go:58]: the same

Comment (jomenxiao) [reply bin/titan/main.go:58]: if `dumpRedisCommand` both setting on configuring file and cmd param. the cmd param will replace configuring file.

File: bin/titan/main.go (hunk 1 above)
Comment (jomenxiao) [reply bin/titan/main.go:38]: the `dumpRedisCommand` is a param what dump's command save directory

**Pull request closed without merging**